src/patternfly/components/Accordion/accordion-expanded-content.hbs

```diff
@@ -1,4 +1,4 @@
-<{{#if accordion--IsDefinitionList}}dd{{else}}div{{/if}} class="pf-c-accordion__expanded-content{{#if accordion-expanded-content--IsExpanded}} pf-m-expanded{{/if}}{{#if accordion-expanded-content--IsFixed}} pf-m-fixed{{/if}}{{#if accordion-expanded-content--modifier}} {{accordion-expanded-content--modifier}}{{/if}}"
+<{{#if accordion--IsDefinitionList}}dd{{else}}div{{/if}} tabindex=0 class="pf-c-accordion__expanded-content{{#if accordion-expanded-content--IsExpanded}} pf-m-expanded{{/if}}{{#if accordion-expanded-content--IsFixed}} pf-m-fixed{{/if}}{{#if accordion-expanded-content--modifier}} {{accordion-expanded-content--modifier}}{{/if}}"
```

Comment:
Rather than applying `tabindex=0` to every content component, we should apply it only when `accordion-expanded-content--IsFixed` is true. Or alternatively we could apply it in the example itself rather than in the handlebars. @mcoker do you have any preference? Ideally we would only want the tabindex when a scrollbar is present, which may not always be the case when the accordion content `isFixed`, but it might depend on if we anticipate showing an example where `isFixed` is applied but there isn't a scrollbar?

Reply:
@thatblindgeye good question. It's most likely true that if we're using a fixed accordion in core, the content shown will likely be scrollable (with a scrollbar visible), but it isn't always necessarily the case.

Ideally I think we could have a conditional in this partial that tests for the presence of a parameter indicating there is scrollable content:

```
{{#if accordion-expanded-content--HasScrollbar}}
  tabindex="0"
{{/if}}
```

That could be set 1 of a few ways:
1. Individually on expandable content elements that have a scrollbar
1. On the parent `{{#> accordion}}` partial, which will set it to true for all expandable content elements
1. We could tie it to `accordion-expanded-content--IsFixed` by setting `accordion-expanded-content--HasScrollbar=accordion-expanded-content--IsFixed`, which is functionally equivalent to changing the conditional above to test for `accordion-expanded-content--IsFixed`. Through some trickery, that could be set by default, then we could set `accordion-expanded-content--HasScrollbar=reset` on individual sections that don't have a scrollbar.

The downside to 1 and 2 is that it requires us to set the tabindex manually per accordion section, and it isn't guaranteed we'll add it consistently. Given that accordions in core are static, it's most likely a fixed demo/example will have scrollable content, so I'd opt for 3. And maybe if we're showing a fixed accordion with multiple sections expanded and not all have scrollbars, we'll have the option to remove the `tabindex`.